### DISCUSSION TITLE
Can i upload files? What type of files can I upload?

### DISCUSSION THREAD

**Discussion by ux-han** [janhq/jan#3525] [Get Help] [answered]
File search? RAG?

**Comment by ux-han** [comment, marked as answer]
File search (RAG) is experimental. Turn on Experimental Model in Settigns > Advanced Settings > Exeperimental Mode. Currently it only supports PDF format. We will improve it soon!

<img width="1105" alt="Screenshot 2024-09-02 at 4 28 47 PM" src="https://github.com/user-attachments/assets/580cf872-5456-48b6-875c-3190a54511f4">

**Reply by Boscop** [reply]
Thanks 🙏

Please also allow attaching images by pasting from clipboard 😊
